metrics-clojure-ring/src/metrics/ring/instrument.clj

```diff
@@ -56,7 +56,7 @@
            (mark-in! schemes request-scheme)
            (let [resp (time! (times request-method (times :other))
                              (handler request))
-                 status-code (or (:status resp) 404)]
+                 ^int status-code (or (:status resp) 404)]
```

Comment:
This should be `^{:tag "int"}`, see [this Eastwood issue](https://github.com/jonase/eastwood/issues/171).